src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export class Actions {
```

Comment:
I see, I didn't realize you were using the action values as labels.

I think that you could replace the labels in the `<ion-segment-button>`s with hardcoded strings, because they are strictly presentational - no logic or dynamic data involved at all. If you did that, then an `enum` should work.

But, if you don't like that approach, it's no big deal - there is no strong advantage to using an `enum`, it just feels like the better solution.